### ISSUE TITLE
Offer nuget package with test cases

### ISSUE THREAD

**Issue by AArnott** [eiriktsarpalis/typeshape-csharp#34]
Your examples' tests have a massive battery of test cases, which is awesome and useful for serialization libraries built on TypeShape outside your repo. Can you offer a separate nuget package that just contains your test cases for consumption by our test projects?

**Comment by eiriktsarpalis** [comment]
Yeah, it's something I've actually been thinking about. It sucks having to replicate all those.
